.vib/elasticsearch/cypress/cypress/integration/utils.js

```diff
@@ -0,0 +1,7 @@
+/// <reference types="cypress" />
+
+export let random = (Math.random() + 1).toString(36).substring(7);
+
+export let randomNumber = (Math.floor(Math.random() * 10000) + 10000)
```

Comment:
Would it be possible to only use one `random` function? It seems they are fairly similar, I think we can work with an random string containing only numbers for the whole test suite.